README.md

```diff
@@ -7,8 +7,8 @@ In order to evaluate quality of ML models, as well as to create “ImageNet for
 ### Docker
 
 ```bash
-docker build -t code-annotation .
-docker run --rm -p 8080:8080 code-annotation
+docker build -t src-d/code-annotation .
```

Comment:
But this is the name of the image when created locally.
Is this really required by https://github.com/src-d/devrel/issues/1 ?

Reply:
Sure. I mean that our docker name is src-d and we suppose that there is not difference between a local image and the dockerhub's.

Reply:
fair enough